pages/oss.tsx

```diff
@@ -0,0 +1,299 @@
+import { Box, DarkMode, Flex, Icon, SimpleGrid } from "@chakra-ui/react";
+import { HomepageFooter } from "components/footer/Footer";
+import { Aurora } from "components/homepage/Aurora";
+import { HomepageTopNav } from "components/product-pages/common/Topnav";
+import { HomepageSection } from "components/product-pages/homepage/HomepageSection";
+import { useTrack } from "hooks/analytics/useTrack";
+import { PageId } from "page-id";
+import { ReactNode } from "react";
+import { BsGithub } from "react-icons/bs";
+import { Heading, LinkButton, Text, TrackedLink } from "tw-components";
+import { ThirdwebNextPage } from "utils/types";
+
+const contributors = [
```

Comment:
can we pull these from the github API directly?

Reply:
ideally during build & on a 1h stale time (so it gets refreshed every 1hr from server-side)

Reply:
Just added it in a getStaticProps, revalidate of 3600, lmk what u think of it